src/Npgsql/Internal/Converters/Networking/NpgsqlCidrConverter.cs

```diff
@@ -0,0 +1,22 @@
+using NpgsqlTypes;
+
+// ReSharper disable once CheckNamespace
+namespace Npgsql.Internal.Converters;
+
+sealed class NpgsqlCidrConverter : PgBufferedConverter<NpgsqlCidr>
+{
+    public override bool CanConvert(DataFormat format, out BufferRequirements bufferRequirements)
+        => CanConvertBufferedDefault(format, out bufferRequirements);
+
+    public override Size GetSize(SizeContext context, NpgsqlCidr value, ref object? writeState)
+        => NpgsqlInetConverter.GetSizeImpl(context, value.Address, ref writeState);
+
+    protected override NpgsqlCidr ReadCore(PgReader reader)
+    {
+        var (ip, netmask) = NpgsqlInetConverter.ReadImpl(reader, shouldBeCidr: true);
+        return new(ip, netmask);
+    }
+
+    protected override void WriteCore(PgWriter writer, NpgsqlCidr value)
+        => NpgsqlInetConverter.WriteImpl(writer, (value.Address, value.Netmask), isCidr: false);
```

Comment:
shouldn't it be `isCidr: true`

Reply:
@roji ? https://github.com/NinoFloris/npgsql/pull/2/files#diff-ddd267c3e808c3a62d0f8a4ada4c8636333d731a857eb009c3b2d9acc21f22feR22

Reply:
Yeah, looks like a mistake - submitted #5338, thanks @MaceWindu. It seems that this is actually ignored on the PG side (there's a comment that says so at least), but definitely good to correct this.